crates/swc_ecma_transforms_compat/tests/optional-chaining/issue-7561/output.js

```diff
@@ -0,0 +1,14 @@
+var _x = /*#__PURE__*/ new WeakMap();
+class Foo {
+    test() {
+        var _this;
+        var _this_y, _this_y1;
+        _this_y === null || _this_y === void 0 ? void 0 : _class_private_field_get(_this_y1 = (_this = this) === null || _this === void 0 ? void 0 : _this.y, _x);
```

Comment:
I'm not sure where our miscommunication is coming from, but this is still incorrect.
```diff
- _this_y === null || _this_y === void 0 ? void 0 : _class_private_field_get(_this_y1 = (_this = this) === null || _this === void 0 ? void 0 : _this.y, _x);
+ (_this = this) === null || _this === void 0 ? void 0 : _class_private_field_get(_this.y, _x);
```